script.js

```diff
@@ -0,0 +1,75 @@
+  const myNavbar = document.querySelector('.main-nav');
+  myNavbar.classList.add('active');
+  const myLogo = document.querySelector('.my-logo');
+  myLogo.classList.add('active');
+  const lineBar = document.querySelector('.hamburger');
+  lineBar.classList.add('active');
+  const imgCross = document.querySelector('.cross-img');
+  imgCross.classList.add('active');
+});
+
+const popOutMenu = document.querySelector('.cross-img');
+popOutMenu.addEventListener('click', () => {
+  const myHeader = document.querySelector('.main-header');
+  myHeader.classList.remove('active');
+  const myNavbar = document.querySelector('.main-nav');
+  myNavbar.classList.remove('active');
+  const myLogo = document.querySelector('.my-logo');
+  myLogo.classList.remove('active');
+  const lineBar = document.querySelector('.hamburger');
+  lineBar.classList.remove('active');
+  const imgCross = document.querySelector('.cross-img');
+  imgCross.classList.remove('active');
+});
+
+const popOutMenu2 = document.querySelector('.link-3');
+popOutMenu2.addEventListener('click', () => {
+  const myHeader = document.querySelector('.main-header');
+  myHeader.classList.remove('active');
+  const myNavbar = document.querySelector('.main-nav');
+  myNavbar.classList.remove('active');
+  const myLogo = document.querySelector('.my-logo');
+  myLogo.classList.remove('active');
+  const lineBar = document.querySelector('.hamburger');
+  lineBar.classList.remove('active');
+  const imgCross = document.querySelector('.cross-img');
+  imgCross.classList.remove('active');
+});
+
+const popOutMenu3 = document.querySelector('.link-4');
+popOutMenu3.addEventListener('click', () => {
+  const myHeader = document.querySelector('.main-header');
+  myHeader.classList.remove('active');
+  const myNavbar = document.querySelector('.main-nav');
+  myNavbar.classList.remove('active');
+  const myLogo = document.querySelector('.my-logo');
+  myLogo.classList.remove('active');
+  const lineBar = document.querySelector('.hamburger');
+  lineBar.classList.remove('active');
+  const imgCross = document.querySelector('.cross-img');
+  imgCross.classList.remove('active');
+});
+
+const popOutMenu4 = document.querySelector('.link-5');
+popOutMenu4.addEventListener('click', () => {
+  const myHeader = document.querySelector('.main-header');
+  myHeader.classList.remove('active');
+  const myNavbar = document.querySelector('.main-nav');
+  myNavbar.classList.remove('active');
+  const myLogo = document.querySelector('.my-logo');
+  myLogo.classList.remove('active');
+  const lineBar = document.querySelector('.hamburger');
+  lineBar.classList.remove('active');
+  const imgCross = document.querySelector('.cross-img');
+  imgCross.classList.remove('active');
+});
+
+/* let navText = document.querySelector("hamburger");
+navText.addEventListener("click",()=> {
+    let myTextStyle = document.querySelectorAll(".link-1", "#main-nav");
+    myTextStyle.classList.add("active");
+}) */
```

Comment:
Hi @Shubh-Dev 👋🏻, I love the overall implementation of the mobile menu. It is beautiful, clean and looks great ❤️. You have done a really nice job here 🎉!

- [ ] My only comment is about the unused block of code. It would be nice if you just deleted to keep the code-base clean and easy to navigate.